upup/models/cloudup/resources/addons/kube-dns.addons.k8s.io/v1.6.0.yaml.template

```diff
@@ -41,12 +41,11 @@ spec:
           - /cluster-proportional-autoscaler
           - --namespace=kube-system
           - --configmap=kube-dns-autoscaler
-          - --mode=linear
           # Should keep target in sync with cluster/addons/dns/kubedns-controller.yaml.base
           - --target=Deployment/kube-dns
           # When cluster is using large nodes(with more cores), "coresPerReplica" should dominate.
           # If using small nodes, "nodesPerReplica" should dominate.
-          - --default-params={"linear":{"coresPerReplica":256,"nodesPerReplica":16,"min":2}}
```

Comment:
The old --default-params won't break cluster-proportional-autoscaler. But it would be better to keep it in sync with upstream.